spec/api/commentable_spec.rb

```diff
@@ -125,11 +125,11 @@ def test_unicode_data(text)
       subject { post "/api/v1/#{commentable_id}/threads", parameters }
 
       shared_examples_for 'CommentThread creation API' do |context='course'|
-        it 'creates a new CommentThread' do
+        it 'creates a new CommentThread and marks it as read for owner user' do
           expect(CommentThread.count).to eq 0
 
           body = parse(subject.body)
-          expect(body).to include('read' => false,
+          expect(body).to include('read' => true,
```

Comment:
@robrap There are cases (comment POST endpoints); where the returned hash is of comment and not thread. In such cases, we need to make another call to get thread details to assert "read" status. 

I have updated `test_thread_marked_as_read` in a way that it will get thread details via GET api and assert `read=true`. It seems like a better approach instead of directly calling on db values.